File: result/translate/transcoder/python2java/java/MINIMUM_PRODUCT_K_INTEGERS_ARRAY_POSITIVE_INTEGERS.java
public static int f_gold ( int [ ] arr , int n , int k ) {
  int [ ] pq = new int [ n ] ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) {
    heapify ( pq , arr [ i ] ) ;
  }
  int count = 0 ;
  int ans = 1 ;
  while ( pq != null && count < k ) {
    ans += heapify ( pq ) ;
    count ++ ;
  }
  return ans ;
}

File: dataset/python2java/python/MINIMUM_PRODUCT_K_INTEGERS_ARRAY_POSITIVE_INTEGERS.py
def f_gold(arr, n, k):
    pq = []
    for i in range(n):
        heapq.heappush(pq, arr[i])
    count = 0
    ans = 1
    while pq and count < k:
        ans += heapq.heappop(pq)
        count += 1
    return ans